api/v1/testkube.yaml

```diff
@@ -33,8 +33,210 @@ tags:
     description: "Executor operations"
   - name: webhook
     description: "Webhook operations"
+  - name: test-triggers
+    description: "Test Triggers CRUD operations"
+  - name: keymap
+    description: "Keymap for fields in a form"
 
 paths:
+  /keymap/triggers:
+    get:
+      tags:
+        - keymap
+        - api
+      summary: Test triggers keymap
+      description: Returns a keymap (supported/allowed fields) for the test trigger UI screen
+      operationId: getKeyMap
+      responses:
+        200:
+          description: successful get operation
+          content:
+            application/json:
+              schema:
+                type: object
+                items:
+                  $ref: "#/components/schemas/TestTriggerKeyMap"
+            text/yaml:
+              schema:
+                type: string
```

Comment:
Is it always successful?

Reply:
Yes, nothing can throw an error in that call.
Only way it can error is if the server is down or under heavy heavy pressure and timeouts, but that is not directly related to this endpoint